server/src/main/java/org/activityinfo/ui/client/page/entry/location/LocationSearchPresenter.java

```diff
@@ -94,10 +94,10 @@ private void numberLocations(List<LocationDTO> locations) {
         int number = 0;
         for (LocationDTO location : locations) {
             if (location.hasCoordinates()) {
-                location.setMarker(String.valueOf((char) ('A' + number)));
                 number++;
+                location.setMarker(number.toString());
```

Comment:
I think the reason this was limited to 26 is that the markers are only large enough to display a single character: one option would be to just use a symbol for the first 26 markers and leave the rest blank.

I think the results are also filtered on the server side, so you'll need to check there as well. Also, right now the dialog won't show any results if there are more X results, we probably want to change this to always show the _first_ X results, even if we cut off the list at 200-300.

You can test these changes locally by running `./gradlew appengineRun` and logging in as `test@test.org` / `testing123`.